### PR TITLE
Testcase for static array with zero array length

### PR DESCRIPTION
**Pull request by ahangsu**
Fixes https://github.com/algorand/js-algorand-sdk/issues/497

### CODE REVIEW

**Review by michaeldiamant — APPROVED** [review]
@ahangsu Thanks for adding a test case here!  As intended, I confirmed the added test case fails in downstream SDKs prior to SDK source changes.

File: features/unit/abijson.feature [hunk 1]
```diff
@@ -16,6 +16,7 @@ Feature: AbiJson
       | add(uint32,uint16)uint32                         | addMethodUint16.json     | abi_responsejsons |
       | referenceTest(account,application,asset)uint8[3] | referenceTestMethod.json | abi_responsejsons |
       | txnTest(txn,pay,keyreg,acfg,axfer,afrz,appl)bool | txnTestMethod.json       | abi_responsejsons |
+      | nonsense((uint64,address)[0])(uint64,address)    | nonsense.json            | abi_responsejsons |
```
Comment (bbroder-algo) [inline features/unit/abijson.feature:19]: probably could do better about the naming convention on these jsons.

Comment (ahangsu) [reply features/unit/abijson.feature]: just changed for a better name

File: features/unit/abi_responsejsons/zeroLenStaticArrayArg.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{"name":"nonsense","args":[{"type":"(uint64,address)[0]"}],"returns":{"type":"(uint64,address)"}}
```
Comment (michaeldiamant) [inline features/unit/abi_responsejsons/zeroLenStaticArrayArg.json:1]: @ahangsu Checking my understanding:  Should the expected _name_ change based on https://github.com/algorand/algorand-sdk-testing/pull/250/commits/3f1fbd8d9c5b5e8037f008324aa9a7e21c62dc8e?

A more robust sanity check is to re-trigger a downstream SDK build + link successful build.


```suggestion
{"name":"zeroLenStaticArrayArg","args":[{"type":"(uint64,address)[0]"}],"returns":{"type":"(uint64,address)"}}
```

Comment (ahangsu) [reply features/unit/abi_responsejsons/zeroLenStaticArrayArg.json:1]: right, the change is correct, and I will locally confirm the change passes

Comment (ahangsu) [reply features/unit/abi_responsejsons/zeroLenStaticArrayArg.json]: Confirmed that the change passes in go-sdk on my local end

Comment (michaeldiamant) [reply features/unit/abi_responsejsons/zeroLenStaticArrayArg.json]: @ahangsu I trust you, but want to verify.  I kicked off one build:  https://app.circleci.com/pipelines/github/algorand/go-algorand-sdk/612/workflows/776b4e2d-c12f-45b2-a788-be1fb5ed32d1.  I'll approve after it completes.

Comment (ahangsu) [reply features/unit/abi_responsejsons/zeroLenStaticArrayArg.json]: got it, was wondering why go SDK CI is still running right now

**Comment by ahangsu** [comment]
weird, cla hangs

**Review by michaeldiamant — APPROVED** [review]

**Pull request merged**